openvdb/Platform.h

```diff
@@ -186,6 +186,12 @@
         _Pragma("message(\"NOTE: ignoring deprecation warning\")")
     #define OPENVDB_NO_DEPRECATION_WARNING_END \
         _Pragma("GCC diagnostic pop")
+#elif defined _MSC_VER
+    #define OPENVDB_NO_DEPRECATION_WARNING_BEGIN \
+        __pragma(warning(push)) \
+        __pragma(warning(disable : 4996))
```

Comment:
Does MSVC have a message pragma?  I think we decided that we want to know, if possible, when a deprecation warning is being suppressed.

Reply:
Yes it does.  I'll look into verifying the syntax.